pysoa/server/server.py

```diff
@@ -174,7 +174,11 @@ def handle_next_request(self):
         self._idle_timer.stop()
         self._idle_timer = None
 
-        PySOALogContextFilter.set_logging_request_context(request_id=request_id, **job_request['context'])
+        try:
+            PySOALogContextFilter.set_logging_request_context(request_id=request_id, **job_request['context'])
+        except TypeError:
+            # non unicode keys in job_request['context'] will break keywording of a function call
+            pass
```

Comment:
Don't pass. Please do this:

```
except TypeError:
    PySOALogContextFilter.set_logging_request_context(
        request_id=request_id, 
        **(six.text_type(k): v for k, v in six.iteritems(job_request['context'])),
    )
```

I would greatly prefer not to lose this logging information.